### ISSUE TITLE
Some  problems about MxNet amp performance.

### ISSUE THREAD

**Issue by wzzju** [apache/incubator-mxnet#19052]
## Description
According to [the mxnet amp official doc](https://github.com/apache/incubator-mxnet/blob/master/docs/python_docs/python/tutorials/performance/backend/amp.md), I execute the same code on Tesla V100( 16GB, single card). However, I can not get 60% speed increase, and it is only about  30% increase, as shown below. I'm not sure whether my experiment configuration is incorrect or not. Could you please give me some suggestions?

### FP32
![image](https://user-images.githubusercontent.com/17102274/91723737-d0f92280-ebce-11ea-89ec-1e1f6dfdcdf7.png)

### AMP
![image](https://user-images.githubusercontent.com/17102274/91723575-85df0f80-ebce-11ea-8347-3ae150b75a34.png)

**Comment by szha** [comment]
@wzzju thanks for reporting. could you also share your workload?

**Comment by sxjscience** [comment]
I believe there is another issue that is related: https://github.com/apache/incubator-mxnet/issues/17665

**Comment by wzzju** [comment]
![image](https://user-images.githubusercontent.com/17102274/92449845-3790c880-f1ed-11ea-95a6-6f748e797e87.png)
I find the MxNet used in NGC 20.06 container is customized，because there is no `F.BatchNormAddRelu` found in the [incubator-mxnet](https://github.com/apache/incubator-mxnet) repo. Besides, `nn.BatchNorm` in NGC 20.06 container MxNet version is also different from this repo, as described below.
![image](https://user-images.githubusercontent.com/17102274/92457617-02897380-f1f7-11ea-97e4-50a7efa8ec75.png)
 The left is in  NGC 20.06 container mxnet version, and the right is in mxnet 1.5 official version.

Why does this happen?

**Comment by wzzju** [comment]
> @wzzju thanks for reporting. could you also share your workload?
Thanks. Tesla V100-16GB and the single card is used.
![image](https://user-images.githubusercontent.com/17102274/92458533-1b465900-f1f8-11ea-9629-fc89c28aa805.png)
![image](https://user-images.githubusercontent.com/17102274/92458844-8728c180-f1f8-11ea-8e48-d02572d67a2f.png)
